src/main/resources/application.yml

```diff
@@ -6,4 +6,12 @@ api:
     url: ${API_ERO_MANAGEMENT_URL}
 
 dynamodb:
-  print-details-table-name: ${DYNAMODB_PRINT_DETAILS_TABLE_NAME}
\ No newline at end of file
+  print-details-table-name: ${DYNAMODB_PRINT_DETAILS_TABLE_NAME}
+  scheduler-lock-table-name: ${DYNAMODB_SCHEDULER_LOCKS_TABLE_NAME}
+
+jobs:
+  print-requests:
+    name: "BatchPrintRequests"
+    cron: ${JOB_PRINT_REQUESTS_CRON}
```

Comment:
As per the earlier comments, did we agree that this property (`cron`) would be a value in this yml, rather than a property from the environment? eg:
```
jobs:
  print-requests:
    name: "BatchPrintRequests"
    cron: 0 0 */1 * * *
```
And then you can do away with the environment variable altogether (which now doesnt match the env name in the readme ?)
My preference here would be to simply put the cron here, then get rid of the env var from the readme 👍

Reply:
@NathanRussellValtech  I have it like this: 
```
cron: "${JOB_PRINT_REQUESTS_CRON:0 0 */1 * * *}" # every hour by default
```

I think it's still worth having a variable to override in the env but with a default value. Do you want me to get rid of the env var completely?